WebHostLib/static/assets/gameInfo/en_Timespinner.md

```diff
@@ -0,0 +1,28 @@
+# Timespinner
+
+## Where is the settings page?
+The player settings page for this game is located <a href="../player-settings">here</a>. It contains all the options
+you need to configure and export a config file.
+
+## What does randomization do to this game?
+Items which the player would normally acquire throughout the game have been moved around. Logic remains, so the game
+is always able to be completed, but because of the item shuffle the player may need to access certain areas before
+they would in the vanilla game. All rings and spells are also randomized into those item locations, therefor you can nolonger craft them at the alchemist
+
+## What is the goal of Timespinner when randomized?
+The goal remains unchanged. Kill the Sandman\Nightmare!
+
+## What items and locations get shuffled?
+All main inventory items, orbs, collectables, and familiers can be shuffled, and all locations in the game which could
+contain any of those items may have their contents changed.
+
+## Which items can be in another player's world?
+Any of the items which can be shuffled may also be placed into another player's world. It is possible to choose to
+limit certain items to your own world.
+
+## What does another world's item look like in Subnautica?
+Items belonging to other worlds are represented by the vanilla item [Elemental Beads](https://timespinnerwiki.com/Use_Items), Elemental Beads have no use in the randomizer
```

Comment:
What an appropriate item for Archipelago, good choice!

Reply:
`What does another world's item look like in Subnautica?` Subnautica?

Reply:
I missed that, haha, woah. 🍝 Some pasta.